### ISSUE TITLE
training loss is nan

### ISSUE THREAD

**Issue by wulongjian** [Zhongdao/Towards-Realtime-MOT#211]
Hello, Thanks for your wonderful works. I meet some problems when I training with my custom dataset, the loss is nan, Is this normal, if not, could u give me some advice?
![image](https://user-images.githubusercontent.com/26458219/101483452-234a9c00-3993-11eb-9110-c44511323b1f.png)

**Comment by qiumei1101** [comment]
hi, did you solve that?

**Comment by 0x45f** [comment]
i meet same problem

**Comment by AlekseySh** [comment]
Any news?
